webapp/channels/src/components/suggestion/suggestion_list.tsx

```diff
@@ -75,37 +89,38 @@ export default class SuggestionList extends React.PureComponent {
     }
 
     updateMaxHeight = () => {
-        if (!this.props.inputRef?.current) {
+        if (!this.props.inputRef) {
             return;
         }
 
-        const inputHeight = this.props.inputRef.current.clientHeight ?? 0;
+        //const inputElement = (this.props.inputRef as React.RefObject<HTMLInputElement>).current;
+        const inputHeight = (this.props.inputRef as React.RefObject<HTMLInputElement>).current!.clientHeight ?? 0;
 
         this.maxHeight = Math.min(
             window.innerHeight - (inputHeight + Constants.POST_MODAL_PADDING),
             Constants.SUGGESTION_LIST_MAXHEIGHT,
         );
 
         if (this.contentRef.current) {
-            this.contentRef.current.style['max-height'] = this.maxHeight;
+            this.contentRef.current.style.maxHeight = `${this.maxHeight}px`;
```

Comment:
Not sure if `px` is the correct unit

Reply:
Yeah, that should be right. The `clientHeight` and `innerHeight` that are used to calculate this are definitely pixel values, so I'd expect this one to be too